### ISSUE TITLE
arXiv:1909.02487

### ISSUE THREAD

**Issue by refraction-ray** [quclub/Paper-reading#55]
* [Link](https://arxiv.org/pdf/1909.02487.pdf)
* Title: Ab-Initio Solution of the Many-Electron Schr¨odinger Equation with Deep Neural
Networks
* Keywords (optional):  
* Authors (optional): David Pfau, James S. Spencer, and Alexander G. de G. Matthews, and W. M. C. Foulkes
* Reason (optional): 
* Summary (optional): 
* Ideas (optional):   So many ideas emerged recently in our study has already been utilized in this work. Emmm, hard to say my feeling. Things include GPU accelerated MC, natural gradient as the same technique with SR in VMC, stochastic update with very few steps in each parameter update, and energy or gradient clipping. It will be very hard to beat google in this field: interplay between ML and other traditional science. That's because they have teams of expert knowing every bits of the most advanced ML work and tricks. Besides, they are now recruiting lots of PhD in physics and chemistry together with the collaboration of professors in universities. Plus their state-of-the-art hardware spec in terms of GPU cluster. Mission impossible.
* Inspirations: 
  - take more input parameters beyond simple spin configurations, this may help and do some tasks supposed to be done in the net.

**Comment by refraction-ray** [comment]
## Relevant work on NN representation of fermionic wavefunctions

* [Deep neural network solution of the electronic Schrödinger equation](https://arxiv.org/pdf/1909.08423.pdf)
This work is less brute-force comparing to the above work by DM. It can be run on single 1080Ti and the converge time is less. However, the author utilized adam as the optimizer which I highly doubt whether the optimization works well. From our experience, simple gradient descent is not very suitable for optimization of general VMC. Instead, we need SR or NG in ML language to quickly converge. For a comparison, see Fig2 in DM's paper.
